style.css

```diff
@@ -995,3 +995,74 @@ textarea[type="text"]::placeholder {
     background-color: transparent;
   }
 }
+
+.ham {
+  display: block;
+  background-color: white;
+  width: 25px;
+  height: 3px;
+  margin: 3px auto;
+}
+
+@media (max-width: 768px) {
+  .navlinks {
+    display: block;
+  }
+
+  .hamburger {
+    display: block;
+
+    /* margin-left:-150%; */
```

Comment:
- [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile:

Reply:
all commented codes are removed from css file.clean code available